elodie/media/media.py

```diff
@@ -287,3 +294,11 @@ def get_class_by_file(cls, _file, classes):
                 return i(_file)
 
         return None
+
+    @classmethod
```

Comment:
Seemed like this was duplicated amongst some of the other classes, so I moved it here. Since it's a classmethod, it will get the expected attribute w/out needing to duplicate the classmethod. For example:

```python
>>> class Foo(object):
...     items = ('a', 'b')
...     @classmethod
...     def get_items(cls):
...         return cls.items
...
>>> class Bar(Foo):
...     items = ('c', 'd')
...
>>> class Baz(Foo):
...     pass
...
>>> Foo.get_items()
('a', 'b')
>>> Bar.get_items()
('c', 'd')
>>> Baz.get_items()
('a', 'b')
```